modules/github_integration/lib/api/v3/github_integration/github_pull_request_representer.rb

```diff
@@ -0,0 +1,166 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2021 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See docs/COPYRIGHT.rdoc for more details.
+#++
+
+require 'roar/decorator'
+require 'roar/json/hal'
+
+module API
+  module V3
+    module GithubIntegration
+      class GithubPullRequestRepresenter < ::API::Decorators::Single
+        include API::Caching::CachedRepresenter
+        include API::Decorators::DateProperty
+        include API::Decorators::LinkedResource
+
+        cached_representer key_parts: %i[github_user merged_by latest_check_runs],
+                           disabled: false
+
+        link :staticPath do
+          next
+        end
+
+        property :id
+
+        property :number,
+                 setter: ->(*) { nil }
+
+        property :github_html_url,
+                 setter: ->(*) { nil }
+
+        property :state,
+                 render_nil: true,
+                 setter: ->(*) { nil }
+
+        property :repository,
+                 render_nil: true,
+                 setter: ->(*) { nil }
+
+        date_time_property :github_updated_at,
+                           render_nil: true,
+                           setter: ->(*) { nil }
+
+        property :title,
+                 render_nil: true,
+                 setter: ->(*) { nil }
+
+        property :body,
+                 render_nil: true,
+                 setter: ->(*) { nil }
```

Comment:
In the scope of OpenProject, every longer text field would have to be a `formattable_property` so that the client gets the raw markdown as well as the html rendered representation of it. I assume that we should be able to render the GitHub flavored markdown just fine as we should support a superset of it.

Reply:
👍  great, will do